testsuite/tests/backtrace/backtrace.byte.reference

```diff
@@ -0,0 +1,26 @@
+a
+b
+Fatal error: exception Backtrace.Error("b")
+Raised at Backtrace.f in file "backtrace.ml", line 11, characters 16-32
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.g in file "backtrace.ml", line 15, characters 4-11
+Re-raised at Backtrace.g in file "backtrace.ml", line 17, characters 62-71
+Called from Backtrace in file "backtrace.ml", line 21, characters 9-25
+Fatal error: exception Backtrace.Error("c")
+Raised at Backtrace.g in file "backtrace.ml", line 18, characters 20-37
+Called from Backtrace in file "backtrace.ml", line 21, characters 9-25
+Fatal error: exception Backtrace.Error("d")
+Raised at Backtrace.f in file "backtrace.ml", line 11, characters 16-32
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.f in file "backtrace.ml", line 11, characters 42-53
+Called from Backtrace.g in file "backtrace.ml", line 15, characters 4-11
+Called from Backtrace in file "backtrace.ml", line 21, characters 9-25
+Fatal error: exception Invalid_argument("index out of bounds")
+Raised at Backtrace in file "backtrace.ml", line 21, characters 12-24
```

Comment:
`backtrace.byte.reference` and `backtrace.reference` appear to be identical, so I don't think we need separate files here.

Reply:
Oops, fixed